src/clj/harja/palvelin/integraatiot/tloik/sanomat/ilmoitustoimenpide_sanoma.clj

```diff
@@ -11,15 +11,15 @@
 (defn muodosta-henkilo [data]
   (when data
     [:henkilo
-     [:etunimi (merkkijono/leikkaa 32 (:etunimi data))]
-     [:sukunimi (merkkijono/leikkaa 32 (:sukunimi data))]
+     [:etunimi (merkkijono/leikkaa 32 (escape-xml-varten (:etunimi data)))]
```

Comment:
```suggestion
     [:etunimi (merkkijono/leikkaa 32 (xml/escape-xml-varten (:etunimi data)))]
```

Reply:
Kiitos fiksistä! Testasin tosiaan vaan ton ytimen liian syvällä replyssä. Tämä pitäis testata ihan kunnolla, koska tieliikenneilmoitukset. 
Ongelma ei koske käyttöliittymää.